server/src/main/java/com/linecorp/centraldogma/server/storage/project/InternalProjectInitializer.java

```diff
@@ -86,17 +91,15 @@ public void initialize(String projectName) {
      */
     public void initialize0(String projectName) {
         final long creationTimeMillis = System.currentTimeMillis();
-        try {
-            executor.execute(createProject(creationTimeMillis, Author.SYSTEM, projectName))
-                    .get();
-        } catch (Throwable cause) {
-            final Throwable peeled = Exceptions.peel(cause);
-            if (peeled instanceof ReadOnlyException) {
-                // The executor has stopped right after starting up.
-                return;
-            }
-            if (!(peeled instanceof ProjectExistsException)) {
-                throw new Error("failed to initialize an internal project: " + projectName, peeled);
+        if (!projectManager.exists(projectName)) {
+            try {
+                executor.execute(createProject(creationTimeMillis, Author.SYSTEM, projectName))
```

Comment:
Suggestion: Could you wrap `createProject()` and `createRepository()` with `forcePush()` command to make it work in `REPLICATION_ONLY` mode?

Reply:
Good suggestion. Applied. 👍